test/source/tests/setup.ts

```diff
@@ -2299,6 +2303,180 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
+        const toEmailHeaderLine = rawMessage.headerLines[3].line;
+        const subjectLine = String(rawMessage.headers.get('subject'));
+        const adminPrivateKey = testConstants.prvBackupToDesignatedMailboxTestPrvKey;
+        const parsedAdminPrivateKey = await KeyUtil.parse(adminPrivateKey);
+        const passphrase = 'super hard to guess passphrase';
+        const encryptedBodyText = rawMessage.text!;
+        const kisWithPassphrase: KeyInfoWithIdentityAndOptionalPp[] = [
+          { ...(await KeyUtil.keyInfoObj(parsedAdminPrivateKey)), family: parsedAdminPrivateKey.family, passphrase },
+        ];
+        const decrypted = await MsgUtil.decryptMessage({
+          kisWithPp: kisWithPassphrase,
+          encryptedData: encryptedBodyText,
+          verificationPubs: [pubkey],
+        });
+        const encryptedAttachmentContent = rawMessage.attachments[0].content;
+        const decryptedAttachmentContent = await MsgUtil.decryptMessage({
+          kisWithPp: kisWithPassphrase,
+          encryptedData: encryptedAttachmentContent,
+          verificationPubs: [pubkey],
+        });
+        expect(fromEmailHeaderLine.includes(acctEmail)).to.equal(true);
+        expect(toEmailHeaderLine.includes(adminEmail)).to.equal(true);
+        expect(subjectLine.includes(acctEmail)).to.equal(true);
+        expect(decrypted.success).to.equal(true);
+        expect(decryptedAttachmentContent.success).to.equal(true);
+      })
+    );
+
+    test(
+      'setup - prv_backup_to_designated_mailbox will take precedence if set with NO_PRV_BACKUP',
+      testWithBrowser(async (t, browser) => {
+        const pubkey = testConstants.prvBackupToDesignatedMailBoxTestPubKey;
+        const acctEmail = 'user@prv-backup-to-designated-mailbox.flowcrypt.test';
+        const adminEmail = 'admin@prv-backup-to-designated-mailbox.flowcrypt.test';
+        /* eslint-disable @typescript-eslint/naming-convention */
+        t.context.mockApi!.configProvider = new ConfigurationProvider({
+          fes: {
+            clientConfiguration: {
+              flags: ['NO_ATTESTER_SUBMIT', 'NO_PRV_BACKUP'],
+              prv_backup_to_designated_mailbox: pubkey,
+              enforce_keygen_algo: 'ecc25519',
+              enforce_keygen_expire_months: 6,
+            },
+          },
+        });
+        /* eslint-disable @typescript-eslint/naming-convention */
+        const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
+        await SetupPageRecipe.createKey(settingsPage, '', 'disabled', {
+          skipForPassphrase: true,
+          backupPrvKeyToMailbox: true,
+          key: {
+            passphrase: 'super difficult to guess passphrase',
+          },
+        });
+        const expectedEmailSubject = `FlowCrypt OpenPGP Private Key backup for user`;
+        const sentMsg = (await GoogleData.withInitializedData(acctEmail)).searchMessagesBySubject(expectedEmailSubject)[0];
```

Comment:
this test is almost the same as `Client configuration prv_backup_to_designated_mailbox test`, let's just check if message with `expectedEmailSubject` was sent, it should be enough here

Reply:
updated. thanks.